### PR TITLE
Release `0.8.0`

### PR DESCRIPTION
**Pull request by gareth-j**
* **Summary of changes**

### Added

- Added option to pass `store` argument to `ModelScenario` init method. [PR #928](https://github.com/openghg/openghg/pull/928)

### Fixed

- Issue caused when passing a list of files to be processed. If OpenGHG had seen some of the files before it would refuse to process any of them - [PR #890](https://github.com/openghg/openghg/pull/890)

### Changed

- Moved to store data in [Zarr](https://github.com/zarr-developers/zarr-python) stores, this should reduce both the size of the object store and memory consumption whilst processing and retrieving data - [PR #803](https://github.com/openghg/openghg/pull/803)
- standardise_footprint was updated to allow a source_format input to be specified. This currently only supports "acrg_org" type but can be expanded upon [PR #914](https://github.com/openghg/openghg/pull/914).
- Internal format for "footprint" data type was updated to rename meteorological variable names to standard names [PR #918](https://github.com/openghg/openghg/pull/918).
- Standardise_footprint now uses the meterological model input as a distinguishing keyword when adding data. [PR #955](https://github.com/openghg/openghg/pull/955).
- Meterological model input renamed from `metmodel` to `met_model` [PR #957](https://github.com/openghg/openghg/pull/957).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,25 @@ All notable changes to OpenGHG will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/openghg/openghg/compare/0.7.1...HEAD)
+## [Unreleased](https://github.com/openghg/openghg/compare/0.8.0...HEAD)
+
+## [0.8.0] - 2024-03-19
+
```
Comment (rt17603) [inline CHANGELOG.md:11]: We should add a quick summary here that is a breaking change and we no longer support object stores generated with earlier versions of openghg.

File: CHANGELOG.md [hunk 2]
```diff
@@ -46,6 +65,8 @@ has not changed. This is to prevent errors when concurrent writes are made to th
 - Datasource UUIDs are no longer stored in the storage class and are now only stored in the metadata store - [PR #752](https://github.com/openghg/openghg/pull/752)
 - Support dropped for Python 3.8 - [PR #818](https://github.com/openghg/openghg/pull/818). OpenGHG now supports Python >= 3.9.
 
+- Updated internal naming and input data_type to use "flux" rather than "emissions" consistently. - [PR #827](https://github.com/openghg/openghg/pull/827)
```
Comment (rt17603) [inline CHANGELOG.md:68]: Looks like this should be moved to under the 0.8.0 section as well?

Comment (rt17603) [reply CHANGELOG.md:68]: This might also be included as a reason this is a breaking change.

**Review by rt17603 — CHANGES_REQUESTED** [review]
Looks good - just a few small things to tidy up in the CHANGELOG.md before this is ready to merge.

File: doc/source/packaging.rst [hunk 3]
```diff
@@ -2,28 +2,26 @@
...
 
 * New features are developed on feature branches, called ``feature-{feature}``,
-  either in the `main Pack and Doc repository <https://github.com/openghg/openghg>`__
-  for authorised developers, or in personal forks for
-  new developers.
+  either in the `devel OpenGHG repository <https://github.com/openghg/openghg>`__
+  for authorised developers.
 * Bug fixes or issue fixes are developed on fix branches, called
-  ``fix-issue-{number}`` (again in either the main repository or forks).
+  ``fix-issue-{number}``.
```
Comment (SutarPrasad) [inline doc/source/packaging.rst:12]: ``Iss{number}-``

File: CHANGELOG.md (hunk 2 above)
Comment (gareth-j) [reply CHANGELOG.md:68]: Ah yeah good catch.

**Review by SutarPrasad — APPROVED** [review]
Looks spectacular, May the force be with you.

**Review by rt17603 — APPROVED** [review]
Happy with the updates to the CHANGELOG.md so I think this is ready to go now.

**Review by brendan-m-murphy — APPROVED** [review]
Looks really quite nice!

**Pull request merged**